src/cryptoadvance/specter/static/styles.css

```diff
@@ -423,7 +446,7 @@ input:hover{
 }
 .note{
     font-size: 0.7em;
-    width: 400px;
+    max-width: 400px;
```

Comment:
Seems like this causes the note on the send screen (showing the available coins and sat/btc conversion to appear a bit strange.

Reply:
hopefully better now, added a few tweak.
In the `.card` element note will be left-aligned.